paddle/fluid/operators/run_program_op.h

```diff
@@ -44,6 +47,9 @@ using BlockDesc = framework::BlockDesc;
 using Variable = framework::Variable;
 using LoDTensor = framework::LoDTensor;
 using SelectedRows = framework::SelectedRows;
+using ExecutorInfo =
+    std::tuple<std::shared_ptr<framework::Executor>,
```

Comment:
maybe we only need one Executor? Executor is stateless and should be able to execute multiple ExecutorPrepareContext

Reply:
Thanks, done